src/helpers/figures/charts/chart_common.ts

```diff
@@ -72,10 +79,19 @@ export function updateChartRangesWithDataSets(
     isStale = true;
     labelRange = range;
   }
+  let title = chartTitle;
+  if (title.type === "reference") {
+    const range = adaptChartRange(title.reference, applyChange);
+    if (range !== title.reference) {
```

Comment:
I'm pretty sure the `if` condition is always true. We compare two Range instance, so the reference is compared and not the actual value of the range. But since `applyChange` returns a copy of the range (that's the part i'm not 100% sure about), the comparison is useless because `range` is always !== `copyOfRange`.

Anyway that's ot something you introduced, so you can juste add a TODO so we will mayeb check if more later.